coalib/output/printers/LogHelper.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import traceback
+
+
+def log_exception(exception, **kwargs):
+    if not isinstance(exception, BaseException):
+        raise TypeError('log_exception can only log derivatives of '
+                        'BaseException.')
```

Comment:
Any ideas why we have this instance check in log_exception? (@shreyans800755, I know that you didn't add this check in this PR)
Probably ``traceback.format_exception`` throws an exception anyway if ``exception`` is not a subclass of BaseException.